manifests/nodejs.yml

```diff
@@ -221,7 +221,27 @@ tests/:
         test_uri.py:
           TestURI: missing_feature
     rasp/:
-      test_lfi.py: missing_feature
+      test_lfi.py:
+        Test_Lfi_BodyJson:
+          '*': *ref_5_24_0
+          nextjs: missing_feature
+        Test_Lfi_BodyUrlEncoded:
+          '*': *ref_5_24_0
+          nextjs: missing_feature
+        Test_Lfi_BodyXml: missing_feature
+        Test_Lfi_Capability: *ref_5_24_0
+        Test_Lfi_Mandatory_SpanTags: *ref_5_24_0
+        Test_Lfi_Optional_SpanTags: *ref_5_24_0
+        Test_Lfi_RC_CustomAction: missing_feature
+        Test_Lfi_StackTrace:
+          '*': *ref_5_24_0
+          nextjs: missing_feature
+        Test_Lfi_Telemetry:
+          '*': *ref_5_24_0
+          nextjs: missing_feature
+        Test_Lfi_UrlQuery:
+          '*': *ref_5_24_0
+          nextjs: missing_feature
```

Comment:
should nextjs work for queyrparams since you implemented the endpoint for it in the weblog ?

Reply:
it fails

Reply:
right, but is it expected to fail ?

Reply:
it seems odd that you implemented the endpoint with querystring but not enable the test no ?

Reply:
is the same for ssrf and sqli, so I haven't reviewed the case.